suites/squid/cephadm/tier-0.yaml

```diff
@@ -0,0 +1,145 @@
+#===============================================================================================
```

Comment:
Change test suite name to specific to test suite operations

Reply:
done